test/conf/commitlog_compression_Zstd.yaml

```diff
@@ -1,2 +0,0 @@
-commitlog_compression:
-    - class_name: ZstdCompressor
```

Comment:
this should be maintained. compression tests should enable both sstable and commitlog compression.